inc/class-wp-sitemaps-stylesheet.php

```diff
@@ -42,24 +42,29 @@ public function render_stylesheet( $type ) {
 	 * @since 5.5.0
 	 */
 	public function get_sitemap_stylesheet() {
-		$css         = $this->get_stylesheet_css();
-		$title       = esc_html__( 'XML Sitemap', 'core-sitemaps' );
-		$description = sprintf(
+		$css           = $this->get_stylesheet_css();
+		$title         = esc_xml( __( 'XML Sitemap', 'core-sitemaps' ) );
+		$sitemaps_link = sprintf(
```

Comment:
@swissspidy I had to pull the `<a>` tag out of the translatable string.  Can you please verify I did that correctly?  Similarly, on line159.